README.md

```diff
@@ -768,10 +773,10 @@ Joins a gateway to the ar.io network via its associated wallet. Requires `signer
 
 ```typescript
 const jointNetworkParams = {
-  qty: 10_000, // minimum operator stake allowed
+  qty: new IOToken(10_000).valueOf(), // minimum operator stake allowed
   autoStake: true, // auto-stake operator rewards to the gateway
   allowDelegatedStaking: true, // allows delegated staking
-  minDelegatedStake: 100, // minimum delegated stake allowed (in mIO)
+  minDelegatedStake: new IOToken(100).valueOf(), // minimum delegated stake allowed (in mIO)
```

Comment:
This line needs the comment changed to `(in IO)`

Reply:
[5d7423a](https://github.com/ar-io/ar-io-sdk/pull/84/commits/5d7423a0a81ea3412c04497644c478a0670ca433)